graphql/appointment/fields.ts

```diff
@@ -173,7 +173,7 @@ export class ExtendedFieldsLectureResolver {
         return await getZoomMeeting(appointment);
     }
     @FieldResolver((returns) => String, { nullable: true })
-    @Authorized(Role.ADMIN, Role.APPOINTMENT_PARTICIPANT, Role.INSTRUCTOR)
+    @Authorized(Role.ADMIN, Role.APPOINTMENT_PARTICIPANT, Role.INSTRUCTOR, Role.TUTOR)
```

Comment:
No, this should be Role.OWNER, not Instructor or Tutor

Reply:
Nice, thanks for the correction!

I've tested it again and still works fine